src/completion.ts

```diff
@@ -40,6 +40,7 @@ You MUST NOT use functions that are not available.`,
         },
       },
     ],
+    function_call: { name: "shell" },
```

Comment:
This was intentional because in local version, I have a few more functions and utilities added to the CLI.